go.mod

```diff
@@ -1,103 +1,114 @@
 module github.com/shipwright-io/operator
 
-go 1.19
+go 1.21
 
 require (
 	github.com/go-logr/logr v1.2.4
 	github.com/manifestival/controller-runtime-client v0.4.0
 	github.com/manifestival/manifestival v0.7.2
-	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.28.0
-	github.com/tektoncd/operator v0.60.0
-	k8s.io/api v0.23.5
-	k8s.io/apiextensions-apiserver v0.23.4
-	k8s.io/apimachinery v0.23.5
-	k8s.io/client-go v0.23.5
-	sigs.k8s.io/controller-runtime v0.11.0
+	github.com/tektoncd/operator v0.68.0
+	k8s.io/api v0.27.4
+	k8s.io/apiextensions-apiserver v0.27.4
+	k8s.io/apimachinery v0.27.4
+	k8s.io/client-go v1.5.2
```

Comment:
Please ensure client-go is aligned with the other k8s versions. This will break things otherwise.

Reply:
Yes there is a replcaement directive for client-go 
k8s.io/client-go => k8s.io/client-go v0.26.8
the k8s.io/client-go v1.5.2 is forced by go mod algorithm
I can see the same thing for the tekton operator go mod https://github.com/tektoncd/operator/blob/main/go.mod#L33

Also, it is worth adding that i tested it locally and it is working correctly